sat/cli/jobstat/main.py

```diff
@@ -24,16 +24,41 @@
 The main entry point for the jobstat subcommand.
 """
 import logging
-from sat.report import Report
-from sat.config import get_config_value
+import textwrap as tr
+
 from sat.apiclient import APIError
 from sat.apiclient.jobstat import JobstatClient
+from sat.report import Report
 from sat.session import SATSession
 
 
 LOGGER = logging.getLogger(__name__)
 
 
+def wrap_long_row_lines(row):
+    """Wrap long lines in the output.
+    Args:
+        wrapper (TextWrapper): A textwrap.TextWrapper object.
+        row (dict): A row in the application data returned
+            from the State Checker.
+    Returns:
+        None. Modifies row in place.
+    """
+    if row.get('apid'):
+        row['apid'] = tr.fill(text=row['apid'], width=20)
+    if row.get('jobid'):
+        row['jobid'] = tr.fill(text=row['jobid'], width=20)
+    if row.get('command'):
+        row['command'] = tr.fill(text=row['command'], width=40)
+    if row.get('node_list'):
+        length = 2
+        nodelist = row['node_list'].split(',')
+        row['node_list'] = '\n'.join(
+            ','.join(e)
+            for e in [nodelist[i : i + length] for i in range(0, len(nodelist), length)]
```

Comment:
nit: pep8/code style issue on this line, there should not be a space after the first `i`.

```suggestion
            for e in [nodelist[i: i + length] for i in range(0, len(nodelist), length)]
```